### ISSUE TITLE
[React Native Gallery Preview]: While navigating between the expanded buttons present in left navigation bar, page is not getting auto scrolled

### ISSUE THREAD

**Issue by anupriya13** [microsoft/react-native-gallery#691]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Keyboard

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Now expand all the button present in left navigation bar
Now try to navigate between the buttons mentioned in above navigation bar using tab key and observe the behavior
Actual Result.
While navigating between the expanded buttons present in left navigation bar , page is not getting auto scrolled and active focused elements are not visible clearly to the user.

Note: Page is getting auto scrolled while using the 'Mouse'

Expected Result:
While navigating between the expanded buttons present in left navigation bar, Page should get auto scrolled in both forward & reverse navigation and active focused element should be visible clearly to the user
Scroll bar should be defined for 'Left navigation bar'

User impact:
Users with motor disability will face difficulties if page is not getting auto scrolled
